samples/wifi/ble_coex/README.rst

```diff
@@ -189,85 +187,95 @@ Build for the nRF5340 DK:
 
 The generated HEX file to be used is :file:`throughput/build/merged.hex`.
 
-Connecting to DKs
-=================
 
-After the DKs are connected to the test PC through USB connectors and powered on, open a suitable terminal, and run the following command:
+Testing
+=======
+
+|test_sample|
+
+1. |connect_kit|
+#. |connect_terminal|
+
+#. Run the following commands to check the available devices:
 
-.. code-block:: console
 
-   $ nrfjprog --com
-   1050043161         /dev/ttyACM0    VCOM0
-   1050043161         /dev/ttyACM1    VCOM1
-   1050724225         /dev/ttyACM2    VCOM0
-   1050724225         /dev/ttyACM3    VCOM1
-   $
+   .. code-block:: console
+
+      $ nrfjprog --com
+      1050043161         /dev/ttyACM0    VCOM0
+      1050043161         /dev/ttyACM1    VCOM1
+      1050724225         /dev/ttyACM2    VCOM0
+      1050724225         /dev/ttyACM3    VCOM1
+      $
 
-In this example, ``1050043161`` is the device ID of the nRF5340 DK and ``1050724225`` is device ID of the nRF7002 DK.
+In this example, ``1050043161`` is the device ID of the nRF5340 DK and ``1050724225`` is the device ID of the nRF7002 DK.
 
 While connecting to a particular board, use the ttyACMx corresponding to VCOM1.
 In the example, use ttyACM1 to connect to the board with device ID ``1050043161``.
 Similarly, use ttyACM3 to connect to the board with device ID ``1050724225``.
 
-.. code-block:: console
+#. Run the following commands to connect to the desired devices:
+
+   .. code-block:: console
 
-   $ minicom -D /dev/ttyACM1 -b 115200
-   $ minicom -D /dev/ttyACM3 -b 115200
+      $ minicom -D /dev/ttyACM1 -b 115200
+      $ minicom -D /dev/ttyACM3 -b 115200
 
 Programming DKs
 ===============
 
-To program the nRF5340 DK:
+Complete the following steps to program the nRF5340 DK:
```

Comment:
You could use tabs for different DKs/devices. For example like in https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/matter/lock/README.html#testing
Like this:
```
.. tabs::

   .. group-tab:: nRF5340 DK

      1. First step
  
   .. group-tab:: nRF7002
     
      1. First step
   ```